sdk/enoki/demo/main.tsx

```diff
@@ -42,6 +42,13 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
 									'705781974144-cltddr1ggjnuc3kaimtc881r2n5bderc.apps.googleusercontent.com',
 							},
 						},
+						windowFeatures: () => {
```

Comment:
Added this, so you can open the oauth flow in a popup instead of a new tab if you want

Reply:
sick af